site/docs/relations/physical_relations.md

```diff
@@ -205,6 +205,23 @@ The streaming aggregate operation leverages data ordered by the grouping express
 | Per Grouping Set | A list of expression grouping that the aggregation measured should be calculated for. | Optional, defaults to 0.                |
 | Measures         | A list of one or more aggregate expressions. Aggregate expressions ordering requirements must be compatible with expected ordering. | Optional, required if no grouping sets. |
 
+## Window Operation
+A window operation is a special type of project operation where every function is a window function and all of the window functions share the same sorting and partitioning. This allows for the sort and partition to be calculated once and shared between the various function evaluations.
+
+| Signature            | Value                                                        |
+| -------------------- | ------------------------------------------------------------ |
+| Inputs               | 1                                                            |
+| Outputs              | 1                                                            |
+| Property Maintenance | Maintains distribution. Eliminates ordering.                 |
+| Direct Output Order  | Same as Project operator (input followed by each window expression). |
+
+### Window Properties
+
+| Property           | Description                     | Required               |
+| ------------------ | ------------------------------- | ---------------------- |
+| Input              | The relational input.           | Required               |
+| Window Expressions | One or more window expressions. | At least one required. |
+
 
 
 ## Hashing Window Operation
```

Comment:
Is this replaced by the above? Seems like maybe it is.

Reply:
Maybe not. It seems the `Window Operator` doesn't need to build hash table.

Reply:
Why not?  Do implementations just sort everything by [ Partition Cols | Order Cols ] and then use a streaming approach?  I suppose this probably makes sense but I haven't implemented window functions so I don't know.

Reply:
Yes. The newly added `Window Operator` will sort by the order columns and then use a streaming approach to calculate. So I think it is different with `Hashing Window Operator ` and `Streaming Window Operator`.

Reply:
This PR was discussed during the community meeting today.  @jacques-n explained the behavior of the "hashing window operator" further.  "hashing" is probably not the correct word to use here.  There is a variant of the window operator that can be used for some window functions (e.g. count) where the function does not need to represent the entire window in memory.

I propose we do not try and solve this problem in this PR.  Perhaps we can replace this whole section:

```
## Hashing Window Operation

A window aggregate operation that will build hash tables for each distinct partition expression.

| Signature            | Value                                                        |
| -------------------- | ------------------------------------------------------------ |
| Inputs               | 1                                                            |
| Outputs              | 1                                                            |
| Property Maintenance | Maintains distribution. Eliminates ordering.                 |
| Direct Output Order  | Same as Project operator (input followed by each window expression). |

### Hashing Window Properties

| Property           | Description                     | Required               |
| ------------------ | ------------------------------- | ---------------------- |
| Input              | The relational input.           | Required               |
| Window Expressions | One or more window expressions. | At least one required. |
```

with:

```
## Hashing Window Operation

A window aggregate operation that does not need to represent the entire window in memory.  TBD.
```

Or, we can just remove the section entirely.

@jacques-n , thoughts?